python/src/trezorlib/protobuf.py

```diff
@@ -269,6 +270,12 @@ def write(self, buf: bytes) -> int:
         return nwritten
 
 
+def get_field_type_object(field: Field) -> Optional[type]:
+    from . import messages
+
+    return getattr(messages, field.type, None)
```

Comment:
there should be ` if not issubclass(thing, (IntEnum, MessageType)): return None` here to avoid accidentally fetching a builtin or something like that